views/view_participant.php

```diff
@@ -0,0 +1,39 @@
+<?php
+authorizedPage();
+include('header.php');
+$button = $_GET['view-id'];
+global $db, $params;
+?>
+<?php
+$result = $db->query("SELECT  participants.participantid, participants.dateofbirth, participants.race, people.firstname, people.lastname, people.middleinit
+FROM participants
+INNER JOIN people ON participants.participantid = people.peopleid WHERE people.peopleid=$button", []);
```

Comment:
Same thing here, use `$1` for the `$button` param and then in the array have `[$button]`. (See comment in database_search.php)